src/index.html

```diff
@@ -13,18 +13,123 @@
 
 </head>
 <body>
-    <nav id="navbar">
-        <div class="navbar-container">
-            <a href="index.html" id="navbar-logo">
-                <img id="nav-logo" src="https://upload.wikimedia.org/wikipedia/commons/9/98/International_Pok%C3%A9mon_logo.svg" alt="Pokemon Logo">
-            </a>
-            <div id="counter-nav">
-                <h2 class="counter-text" id="poke-counter"> Number of Pokemons:</h2>
-                <h2 class="counter-text" id="counter">271</h2>
+    <div class="main">
+        <nav id="navbar">
+            <div class="navbar-container">
+                <a href="index.html" id="navbar-logo">
+                    <img id="nav-logo" src="https://upload.wikimedia.org/wikipedia/commons/9/98/International_Pok%C3%A9mon_logo.svg" alt="Pokemon Logo">
+                </a>
+                <div id="counter-nav">
+                    <h2 class="counter-text" id="poke-counter"> Number of Pokemons:</h2>
+                    <h2 class="counter-text" id="counter">271</h2>
+                </div>
+            </div>
+        </nav>
```

Comment:
I see that you put the nav-bar inside a div along with the rest of the code.
Having two main containers: 1 div and 1 footer. 
I think it would help your code if instead of having the nav-bar inside a div with the rest of the code, you would use the `<header>` and `<main>` tags to differentiate the sections.